src/LoRa.h

```diff
@@ -22,7 +22,8 @@ class LoRaClass : public Stream {
   void end();
 
   int beginPacket(int implicitHeader = false);
-  int endPacket();
+  int endPacket(bool async = false);
+  bool isTransmitting();
```

Comment:
I'd prefer if this API was private for now.

Reply:
How is this meant to be private if the current `endPacket` is public?

Reply:
Does this mean, leave `int endPacket()` in public and move `int endPacket(bool async = false)` private? Don't know C++ classes well enough to understand what this will accomplish.

Reply:
@morganrallen I was thinking `isTransmitting()` should be private for now.

With the current PR`beginPacket(...)`, fails if there is a transmission in progress.

Reply:
@morganrallen any thoughts on the above?

Reply:
Ah right, thanks makes sense, missed this comment last week.